src/parser/wsdl.js

```diff
@@ -362,7 +374,7 @@ class WSDL {
    * By the time file A starts processing its includes its definitions will be already loaded,
    * this is the only thing that B will depend on when "opening" A
    */
-  static load(uri, options, callback) {
+  static load(uri, options, callback, syncLoad) {
```

Comment:
What would be the advantage of having `syncLoad` as a standalone parameter than having it be part of `options`?

Reply:
It is a hidden internal only driven by using the loadSync and not exposed in options. If you want I can place it back into options and remove from the 2 static calls if you want?

Reply:
Again, mixing the implementation of `async` and `sync` style is anti-pattern.

Passing `syncLoad` as an argument after `callback` is anti-pattern too.

Reply:
Ah that's true. Haven't thought about that. Good catch.

Reply:
Understood, I'll create 2 new static methods here loadSync and openSync as below and remove the parameter